### ISSUE TITLE
Where to add Player class in order to get instance of Player everywhere in the code

### ISSUE THREAD

**Issue by ashish2199** [ashish2199/Aidos#34]
So far I feel Player belongs to a scene and should be placed in classes that represent levels like Sandbox class.
Also I feel entities(player, enemies and other things) should be a part of scene only.

What are your opinions ?

**Comment by CoreyHendrey** [comment]
Already posted in the Pull Request, but to reiterate, I think this is a good idea too.
Maybe we could have a seperate class to keep the adding functions in?

Sandbox.EntityListObject.addEntity(e);

Kind of thing? To keep Sandbox clean, but still be able to have more verbose validation on functions like addEntity().

**Comment by ashish2199** [comment]
Yeah we can have something similar. This object can have methods like removeDeadEnemies( ) , addEnemyToScreen( ) , checkEnenmyCollisionWithPlayer( ) 
I think its a good idea to separate such functions to from sandbox class.

**Comment by CoreyHendrey** [comment]
Sounds good, though I think "checkEnenmyCollisionWithPlayer( )" would be inside each enemy, rather than a global check, wouldn't it?

**Comment by ashish2199** [comment]
I feel having it local to entity is better as I feel otherwise the function will be large and complicated as it will have special case for different types of enemies.

**Comment by CoreyHendrey** [comment]
Agreed
